nixos/modules/programs/clickshare.nix

```diff
@@ -0,0 +1,27 @@
+{ config, lib, pkgs, ... }:
+
+let
+
+  clicksharePkg = pkgs.clickshare-csc1;
+
+in
+
+{
+
+  options.programs.clickshare-csc1.enable =
+    lib.options.mkEnableOption "Barco ClickShare CSC-1 driver/client";
+
+  config = lib.modules.mkIf config.programs.clickshare-csc1.enable {
+    environment.systemPackages = [ clicksharePkg ];
+    services.udev.packages = [ clicksharePkg ];
+    users.groups.clickshare =
+      # The package brings a udev rule file with a group name.
+      # This module has to define that group.
+      # We check for a name mismatch here:
+      assert clicksharePkg.usersGroup=="clickshare";
```

Comment:
Why do we check this here?
How can a user of the module redefine `usersGroup` by accident? :confused: